client/src/renderer/components/indicators/active-app-indicator.tsx

```diff
@@ -44,20 +44,21 @@ const apps: { [key: string]: { name: string; icon: string } } = {
 const ActiveAppIndicatorComponent: React.FC<{
   app: string;
   pluginInstalled: boolean;
-}> = ({ app, pluginInstalled }) => {
+  icon: string;
+}> = ({ app, pluginInstalled, icon }) => {
   let name = app;
   if (Object.keys(apps).includes(app)) {
     name = apps[app].name;
   } else if (!pluginInstalled) {
     name = "Other";
   }
 
-  const icon = Object.keys(apps).includes(app) ? apps[app].icon : windowIcon;
+  const iconSrc = icon || (Object.keys(apps).includes(app) ? apps[app].icon : windowIcon);
```

Comment:
`icon` and `iconSrc` represent `src` values, so what about renaming this methods `icon` to `pluginSuppliedIcon` and `iconSrc` to `icon`